src/AppInstallerRepositoryCore/Rest/RestSourceFactory.cpp

```diff
@@ -19,7 +20,8 @@ namespace AppInstaller::Repository::Rest
             {
                 THROW_HR_IF(E_INVALIDARG, !Utility::CaseInsensitiveEquals(details.Type, RestSourceFactory::Type()));
 
-                RestClient restClient = RestClient::Create(details.Arg);
+                HttpClientHelper helper{};
+                RestClient restClient = RestClient::Create(details.Arg, std::move(helper));

```

Comment:
>std::move(helper) [](start = 72, length = 17)

If this is mostly for tests, either make a default value or an overload that doesn't require it.